polylogue/cli/reprocess.py

```
@@ -26,6 +27,19 @@ def run_reprocess(
         Exit code
     """
     console = env.ui.console
+    settings = env.settings
+    outputs = env.config.defaults.output_dirs
+
+    def _output_dir(import_provider: str):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_output_dir`

(ANN202)